<launch>
    <param name="input_topic" value="/velodyne_points" />
    <param name="/x" value="2.6" />
    <param name="/y" value="0.5" />
    <param name="/z" value="0" />
    <param name="/roll" value="0" />
    <param name="/pitch" value="0" />
    <param name="/yaw" value="0.07" />
    <param name="/base_height" value="0.9" />
    <rosparam param="map_list" >[1, 2, 3, 4]</rosparam>

    <param name="max_iter" value="30" />
    <param name="regis_type" value="icp" />
    <param name="translation_epsilon" value="0.01" />
    <param name="step_size" value="0.04" />
    <param name="reso" value="0.08" />
    <node pkg="pcl_multitask" type="localize_indoor_node" name="localize_indoor_node" output="screen" />
    <!--- Run Rviz-->
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find pcl_multitask)/rviz/localize_indoor.rviz" />
</launch>
